pkg/config/config.go

```diff
@@ -216,7 +247,7 @@ func detectCgroupPath(basePath string) string {
 }
 
 func getContainerMemory(memoryValueType memoryValueType) (uint64, error) {
-	if cgroupVersionDetected == noCgroupVersion {
+	if cgroupVersionDetected == 0 {
```

Comment:
Hmm I actually think the way you had it before was more readable. So I'd suggest changing it back

Sorry for my bad advice to remove `noCgroupVersion`

Reply:
No worries!